### ISSUE TITLE
/x dot matches depend on the block size

### ISSUE THREAD

**Issue by codido** [radare/radare2#4205]
```
[0x00000000]> w AAAA
[0x00000000]> /x 41.1
Searching 2 bytes in [0x0-0x200]
hits: 2
0x00000000 hit0_0 4141
0x00000002 hit0_1 4141
[0x00000000]> b 3
[0x00000000]> /x 41.1
Searching 2 bytes in [0x0-0x200]
hits: 1
0x00000000 hit1_0 4141
[0x00000000]> b 2
[0x00000000]> /x 41.1
Searching 2 bytes in [0x0-0x200]
hits: 0
[0x00000000]> /x 4141
Searching 2 bytes in [0x0-0x200]
hits: 2
0x00000000 hit2_0 4141
0x00000002 hit2_1 4141
```

**Comment by radare** [comment]
Add the tests in the r2r repo

**Comment by gitido** [comment]
@radare The 10th test case in t/search/hex ('any-match with small block size')

**Comment by radare** [comment]
ping @jroimartin

**Comment by jroimartin** [comment]
@radare pong. Sorry for the late response... I'll fix it in the next few days

**Comment by jroimartin** [comment]
Should be fixed with #4970. Please check.
